core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -26,11 +26,11 @@ class PerfConfig(args: Array[String]) {
     .withRequiredArg
     .describedAs("count")
     .ofType(classOf[java.lang.Long])
-  val reportingIntervalOpt = parser.accepts("reporting-interval", "Interval at which to print progress info.")
+  val reportingIntervalOpt = parser.accepts("reporting-interval", "Interval in milliseconds at which to print progress info.")
     .withRequiredArg
-    .describedAs("size")
+    .describedAs("interval_ms")
     .ofType(classOf[java.lang.Integer])
-    .defaultsTo(5000)
+    .defaultsTo(50)
```

Comment:
Yes, I'll revert it back to 5000, which implies a 5 second interval with this update.